### PR TITLE
Regenerate model/requests

### PR DESCRIPTION
**Pull request by kevklam**
-UIKit casing, 'recent' support, codegen comment.
-Update bundle ID in iOSExplorer.

### CODE REVIEW

File: Examples/iOSExplorer/iOSExplorer/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -59,6 +59,11 @@
       "idiom" : "ipad",
       "size" : "76x76",
       "scale" : "2x"
+    },
+    {
+      "idiom" : "ipad",
+      "size" : "83.5x83.5",
+      "scale" : "2x"
```
Comment (peternied) [inline Examples/iOSExplorer/iOSExplorer/Images.xcassets/AppIcon.appiconset/Contents.json]: Why is this included?

File: OneDriveSDK/OneDriveCoreSDK/model/ODAsyncOperationStatus.h [hunk 2]
```diff
@@ -18,10 +18,15 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 // 
-// 
-// This file was generated and any changes will be overwritten.
+// CodeGen: b53160c326682c5d0326144548f8f1a5297b0f62
```
Comment (peternied) [inline OneDriveSDK/OneDriveCoreSDK/model/ODAsyncOperationStatus.h:21]: Yay for traceback!

File: OneDriveSDK/OneDriveCoreSDK/requests/ODDriveRequestBuilder.m [hunk 3]
```diff
@@ -70,4 +75,9 @@ - (ODDriveRequest *) requestWithOptions:(NSArray *)options
     return [[ODDriveRequest alloc] initWithURL:self.requestURL options:options client:self.client];
 }
 
+- (ODDriveRecentRequestBuilder *)recent
```
Comment (peternied) [inline OneDriveSDK/OneDriveCoreSDK/requests/ODDriveRequestBuilder.m:78]: FYI - You are including view.recent, might want to quickly gut check this functionality.

**Comment by peternied** [comment]
How did the CI integration with travis miss this?

File: Examples/iOSExplorer/iOSExplorer/Images.xcassets/AppIcon.appiconset/Contents.json (hunk 1 above)
Comment (kevklam) [inline Examples/iOSExplorer/iOSExplorer/Images.xcassets/AppIcon.appiconset/Contents.json:66]: It seems to get auto generated in the latest version of Xcode

File: OneDriveSDK/OneDriveCoreSDK/requests/ODDriveRequestBuilder.m (hunk 3 above)
Comment (kevklam) [inline OneDriveSDK/OneDriveCoreSDK/requests/ODDriveRequestBuilder.m:78]: Seems to work!
